ledger/simulation/trace.go

```diff
@@ -108,8 +108,9 @@ func (eo ResultEvalOverrides) LogicEvalConstants() logic.EvalConstants {
 type ExecTraceConfig struct {
 	_struct struct{} `codec:",omitempty"`
 
-	Enable bool `codec:"enable"`
-	Stack  bool `codec:"stack-change"`
+	Enable        bool `codec:"enable"`
+	Stack         bool `codec:"stack-change"`
+	ScratchChange bool `codec:"scratch-change,omitempty"`
```

Comment:
Why isn't "stack-change" omitempty?

Reply:
oh, good catch, actually `ScratchChange` (now `Scratch`) does not have to be `omitempty`, for all of the fields in the struct are automatically `omitempty` by first line:
```
_struct struct{} `codec:",omitempty"`
```